### PR TITLE
#836 - Full text search IRI not set from Knowledge Base Profile

### PR DESCRIPTION
**Pull request by MdeBoer95**
**What's in the PR**
* Fix for #836 

**How to test manually**
* Create new knowledge base and select a profile with pre-defined full-text-search IRI
* See if full-text-search IRI is set properly

**Automatic testing**
* [ ] PR includes unit tests

**Documentation**
* [ ] PR updates documentation

### CODE REVIEW

File: inception-kb/src/main/resources/de/tudarmstadt/ukp/inception/kb/yaml/knowledgebase-profiles.yaml [hunk 1]
```diff
@@ -130,12 +131,12 @@ wildlife_ontology:
       property-label: http://www.w3.org/2000/01/rdf-schema#label
       property-description: http://www.w3.org/2000/01/rdf-schema#comment
       subproperty-of: http://www.w3.org/2000/01/rdf-schema#subPropertyOf
-      full-text-search: http://www.openrdf.org/contrib/lucenesail#matches
 zbw-stw-economics:
   name: STW Thesaurus for Economics
   type: REMOTE
   access:
       access-url: http://zbw.eu/beta/sparql/stw/query
+      full-text-search: http://www.openrdf.org/contrib/lucenesail#matches
```
[the remark this answers is not in the document]
Comment (MdeBoer95) [reply inception-kb/src/main/resources/de/tudarmstadt/ukp/inception/kb/yaml/knowledgebase-profiles.yaml]: Not sure, I just took the values that were already there. I'll have a look at it.

**Comment by reckart** [comment]
Unrelated question: does the knowledgebase-profiles.yaml already support specifying root concepts?

**Comment by MdeBoer95** [comment]
Yes, see for example Wikidata or DBPedia

File: inception-kb/src/main/resources/de/tudarmstadt/ukp/inception/kb/yaml/knowledgebase-profiles.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (MdeBoer95) [reply inception-kb/src/main/resources/de/tudarmstadt/ukp/inception/kb/yaml/knowledgebase-profiles.yaml]: Looks like it's not supported so I set the IRIs to FTS:NONE

**Pull request merged**